app/jobs/bulk_action_jobs.rb

```diff
@@ -205,7 +205,9 @@ def perform(collection_id, overwrite = true)
         media_object = MediaObject.find(id)
         media_object.hidden = collection.default_hidden
         media_object.visibility = collection.default_visibility
-        media_object.lending_period = collection.default_lending_period
+        if Settings.controlled_digital_lending.enable
```

Comment:
I think we need to handle the cases when `Settings.controlled_digital_lendings.enable` is not set or not a boolean:
```suggestion
        if !!Settings.controlled_digital_lending&.enable
```
If it seems nicer, we could create a convenience method to do the special handling.  Maybe something like this in `lib/avalon/configuration.rb`:
```
# To be called as Avalon::Configuration.controlled_digital_lending_enabled?
module Avalon
  module Config
    def controlled_digital_lending_enabled?
      !!Settings.controlled_digital_lending&.enable
    end
  end
end
```

Reply:
Good call. I think I lean towards using the convenience method, so will try that out.